docs/get-started/index.mdx

```diff
@@ -0,0 +1,241 @@
+Avalonia Window                               avalonia.window             [C#],F#     Desktop/Xaml/Avalonia/Windows/Linux/macOS
+```
+
+You can run this command to check what templates you have available for `dotnet new`:
+
+```
+ dotnet new list
+```
+
+Along with other .NET templates, you should see these Avalonia templates:
+
+```
+Template Name                                 Short Name                  Language    Tags
+--------------------------------------------  --------------------------  ----------  ---------------------------------------------------------
+Avalonia .NET App                             avalonia.app                [C#],F#     Desktop/Xaml/Avalonia/Windows/Linux/macOS
+Avalonia .NET MVVM App                        avalonia.mvvm               [C#],F#     Desktop/Xaml/Avalonia/Windows/Linux/macOS
+Avalonia Cross Platform Application           avalonia.xplat              [C#],F#     Desktop/Xaml/Avalonia/Browser/Mobile
+```
+
+### Installation troubleshooting
+
+#### .NET is not a recognized program
+
+First, ensure the .NET SDK is installed. Run this command:
+
+```
+dotnet --list-sdks
+```
+
+If a .NET SDK is correctly installed, this returns an output similar to the following:
+
+```
+8.0.202 [C:\Program Files\dotnet\sdk]
+```
+
+If the terminal continues to report that .NET is missing, try restarting the terminal.
+
+#### `Avalonia.Templates` package cannot be found
+
+Ensure NuGet is correctly set up. Run this command:
+
+```
+dotnet nuget list source
+```
+
+Check that the output displays the following as a registered source:
+
+```
+nuget.org [Enabled]
+https://api.nuget.org/v3/index.json
+```
+
+If this source is not listed, add it using this command:
+
+```
+dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org
+```
+
+If the package install continues to fail even with NuGet listed as a registered source, check your network connectivity or firewall settings.
+
+## Creating your first project
+
+For your first project, we’re going to use the MVVM template.
+
+<Tabs groupId="ide">  
+  <TabItem value="rider" label="Rider">
+	1. On the Rider startup screen, select **New Solution**.
+  2. In the side bar, scroll down to “Custom Templates”. Select **Avalonia .NET MVVM App**.
+  3. Name your solution “GetStartedApp”.
+  4. Click **Create**.
+  </TabItem>
+  <TabItem value="vs" label="Visual Studio">
+  1. In Visual Studio, click **File → New → Project/Solution**.
+  2. In the search box, input “Avalonia”.
+  3. Select **Avalonia .NET MVVM App** from the search results. If Visual Studio offers multiple options, select the one for **C#**. Then, click **Next**.
+  4. Name your project “GetStartedApp”.
+  5. Change the target directory if desired. Then, click **Next**.
+  6. Select your preferred version of **.NET** as the framework.
+  7. If given the option to choose target platforms, select **Desktop**. Then, click **Next**.
+  8. Select **ReactiveUI** as the toolkit. Then, click **Create**.
```

Comment:
Why make this selection when nothing else in this doc or the starter-tutorial does anything with it?